src/Symfony/Component/Validator/Test/ConstraintValidatorTestCase.php

```diff
@@ -130,16 +130,22 @@ protected function createContext()
         $context->setNode($this->value, $this->object, $this->metadata, $this->propertyPath);
         $context->setConstraint($this->constraint);
 
-        $contextualValidator = $this->getMockBuilder(AssertingContextualValidator::class)
-            ->setConstructorArgs([$context])
-            ->setMethods([
-                'atPath',
-                'validate',
-                'validateProperty',
-                'validatePropertyValue',
-                'getViolations',
-            ])
-            ->getMock();
+        $contextualValidatorMockBuilder = $this->getMockBuilder(AssertingContextualValidator::class)
+            ->setConstructorArgs([$context]);
+        $contextualValidatorMethods = [
+            'atPath',
+            'validate',
+            'validateProperty',
+            'validatePropertyValue',
+            'getViolations',
+        ];
+        // PHPUnit 10 deprecated MockBuild::setMethods()
```

Comment:
```suggestion
        // PHPUnit 10 removed MockBuild::setMethods()
```

The deprecation happened in 9.something already.